### ISSUE TITLE
Create a section in app registry called "Discontinued Apps" and put Rabbithole in it

### ISSUE THREAD

**Issue by bitsikka** [BrightID/BrightID#1171]
Problem: There are members of the public holding BrightID asking how to connect to discontinued apps like Rabbthole.

Simply having a "Discontinued Apps" section and having those apps categorized as such seems obvious. In both app registries:
- the website
- the mobile app

Communicate clearly there the holder cannot link with it anymore and it is being listed for historical record.

**Comment by adamstallard** [comment]
Yes. We need this.  We need to be able to mark apps as "discontinued" so that we can still show the users they sponsored in https://apps.brightid.org , but they won't show up in the apps tab.

I don't think we need a "discontinued apps" section in the mobile app, but we do want one on https://apps.brightid.org org.  In the mobile app, we can just not show the discontinued apps.

**Comment by QuickMythril** [comment]
i support this idea, and would like to suggest MultiverseDAO for the discontinued list as well.  from what i could see, there is no way to sign up and their discord server had no recent activity at all.

**Comment by adamstallard** [comment]
This will also require a change to https://github.com/BrightID/BrightID-Apps-Page  and the format of the spreadsheet it reads.
